lib/pages/Songs_List.dart

```diff
@@ -0,0 +1,140 @@
+                  icon: Icon(Icons.library_music), label: "Library"),
+              BottomNavigationBarItem(
+                  icon: Icon(Icons.person), label: "Library")
+            ]),
+        backgroundColor: Color.fromARGB(148, 0, 0, 0),
+        appBar: AppBar(
+          backgroundColor: Colors.black87,
+          elevation: 0.0,
+          title: Text(
+            "Your Songs (draft)",
+          ),
+        ),
+        body: FutureBuilder<String>(
+          future:
+              DefaultAssetBundle.of(context).loadString("AssetManifest.json"),
+          // future: rootBundle.loadString("AssetManifest.json")
+          builder: (context, item) {
+            if (item.hasData) {
+              Map? jsonMap = json.decode(item.data!);
+              // List? songs = jsonMap?.keys.toList();
+              List? songs = jsonMap?.keys
+                  .where((element) => element.endsWith(".mp3"))
+                  .toList();
+
+              return ListView.builder(
+                itemCount: songs?.length,
+                itemBuilder: (context, index) {
+                  var path = songs![index].toString();
+                  var title = path.split("/").last.toString();
+                  title = title.replaceAll("%20", "");
+                  title = title.split(".").first;
+
+                  return Container(
+                    margin: const EdgeInsets.only(
+                        top: 0.0, left: 10.0, right: 10.0),
+                    decoration: BoxDecoration(
+                      color: Color.fromARGB(185, 21, 40, 54),
+                      borderRadius: BorderRadius.circular(4.0),
+                      border: Border.all(
+                        color: Colors.black,
+                        width: 1.0,
+                      ),
+                    ),
+                    child: ListTile(
+                      trailing: Row(
+                        mainAxisSize: MainAxisSize.min,
+                        children: [
+                          IconButton(
+                            icon: Icon(Icons.download),
+                            color: Colors.white,
+                            onPressed: () {},
+                          ),
+                          IconButton(
+                            icon: Icon(Icons.favorite),
+                            color: Colors.white,
+                            onPressed: () {},
+                          ),
+                          IconButton(
+                            icon: Icon(Icons.more_horiz),
+                            color: Colors.white,
+                            onPressed: () {},
+                          ),
+                        ],
+                      ),
+                      textColor: Color.fromARGB(204, 11, 170, 14),
+                      title: Text(
+                        title,
+                      ),
+                      subtitle:
+                          Text("path: ${path.substring(13, path.length - 4)}",
```

Comment:
Use Subsonic api to fetch songs. 

You migt need to use for loop to iterate through songs of each artists, and then display as songs.